python/lsst/daf/butler/core/storageClassDelegate.py

```diff
@@ -246,8 +246,8 @@ def getComponent(self, composite: Any, componentName: str) -> Any:
         return component
 
     def disassemble(
-        self, composite: Any, subset: Optional[Iterable] = None, override: bool = None
-    ) -> Dict[str, Any]:
+        self, composite: Any, subset: Optional[Iterable] = None, override: Optional[Any] = None
+    ) -> Optional[Dict[str, DatasetComponent]]:
```

Comment:
I can't understand how this ever returns `None`. `components` is always a dict and is returned as a dict unless an exception is raised.

Reply:
I'll try to remove the `Optional` and report back if I encounter a problem.  I don't recall how it got here.

Reply:
I remembered after looking at the code: the (preexisting) docstring says:
> Returns None if this is not a composite `StorageClassDelegate.storageClass`.

I assume I should remove that sentence from the docs, then?

Reply:
I guess it changed at some point. It will seemingly always return an empty dict.